.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+sudo: false
+services:
+  - docker
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+script: bash -ex ./.travis-docker.sh
+env:
+  global:
+  - PACKAGE="xenstore_transport"
+  - PINS="xenstore_transport:. xenstore-tool:."
```

Comment:
Thanks! 
I can see it being really useful with `jbuilder`. This saved me the time to figure out how to make the new Travis files for the ported packages :)